packages/cli/src/__tests__/fixtures/custom-generator/prisma/schema.prisma

```diff
@@ -0,0 +1,34 @@
+generator min {
+  // https://github.com/timsuchanek/minimal-generator
+  provider = "npx @timsuchanek/minimal-generator"
```

Comment:
How does this work?

Reply:
It runs the command, it's what powers the go client like https://github.com/prisma/prisma-client-go/blob/master/docs/quickstart.md
```
generator db {
    provider = "go run github.com/prisma/prisma-client-go"
}
```

Reply:
Source code should be here https://github.com/prisma/prisma/blob/f731467b8ac7169bfc97212abb537f4bbf9ebc92/packages/generator-helper/src/GeneratorProcess.ts#L48-L66